packages/core/src/util/cache.js

```diff
@@ -40,19 +67,37 @@ export function lruCache({
     }
   }
 
+  function get(key) {
+    const entry = cache.get(key);
+    if (entry) {
+      entry.last = performance.now();
+      return entry.value;
+    }
+  }
+
+  function set(key, value) {
+    cache.set(key, { last: performance.now(), value });
+    if (cache.size > max) requestIdle(evict);
+    return value;
+  }
+
   return {
-    get(key) {
-      const entry = cache.get(key);
-      if (entry) {
-        entry.last = performance.now();
-        return entry.value;
-      }
+    get,
+    set,
+    clear() { cache = new Map; },
+    export() {
+      return Array.from(cache).reduce(
+        (acc, [key, entry]) => {
+          acc.set(key, tableToIPC(entry.value, { format: 'stream' }));
+          return acc;
+        },
+        new Map()
+      );
```

Comment:
Here is an alternative, which returns a [key, value] array instead of a `Map`. Also, we don't need to indicate the `stream` IPC format as that is the default.

```js
return Array.from(cache).map([key, entry] => [key, tableToIPC(entry.value)]);
```